disttask/importinto/subtask_executor.go

```diff
@@ -77,8 +78,14 @@ func (e *importMinimalTaskExecutor) Run(ctx context.Context) error {
 	})
 	chunkCheckpoint := toChunkCheckpoint(e.mTtask.Chunk)
 	sharedVars := e.mTtask.SharedVars
-	if err := importer.ProcessChunk(ctx, &chunkCheckpoint, sharedVars.TableImporter, sharedVars.DataEngine, sharedVars.IndexEngine, sharedVars.Progress, logger); err != nil {
-		return err
+	if sharedVars.TableImporter.IsLocalSort() {
+		if err := importer.ProcessChunk(ctx, &chunkCheckpoint, sharedVars.TableImporter, sharedVars.DataEngine, sharedVars.IndexEngine, sharedVars.Progress, logger); err != nil {
+			return err
+		}
+	} else {
+		if err := importer.ProcessChunkWith(ctx, &chunkCheckpoint, sharedVars.TableImporter, dataWriter, indexWriter, sharedVars.Progress, logger); err != nil {
```

Comment:
global sort use ProcessChunkWith, but local use ProcessChunk

Reply:
that's expected

Reply:
I mean the name of function is not consistency.